Makefile

```diff
@@ -0,0 +1,11 @@
+clean:
+	git clean -fxd
+
+install:
+	npm install
+	obt install --verbose
+
+verify:
+	obt verify
+
+test: verify
```

Comment:
The tests are in the circle.yml. I've got no idea how this Makefile got here, I'll remove it